src/jit/lsra.cpp

```diff
@@ -8450,7 +8424,9 @@ void LinearScan::resolveRegisters()
         printf("Trees after linear scan register allocator (LSRA)\n");
         compiler->fgDispBasicBlocks(true);
     }
-    verifyFinalAllocation();
+
+// TODO(pdg): re-enable verification once it's moved to LIR, as it will almost certainly be useful.
+// verifyFinalAllocation();
```

Comment:
The verification and the dump are enabled with the latest changes--would it be helpful if I are based again in order to fold them in with this commit?